EDAM_dev.owl

```diff
@@ -33181,7 +33181,7 @@
         <created_in>1.11</created_in>
         <documentation rdf:resource="http://seqanswers.com/wiki/AFG"/>
         <oboInOwl:hasDbXref rdf:resource="http://seqanswers.com/wiki/AFG"/>
-        <oboInOwl:hasDefinition>AFG is a single text-based file assembly format  that holds read and consensus information together</oboInOwl:hasDefinition>
+        <oboInOwl:hasDefinition>AFG is a single text-based file assembly format hat holds read and consensus information together</oboInOwl:hasDefinition>
```

Comment:
One 't' too little 😊

Reply:
Oops 😅 Thanks

Reply:
It should be fixed now 😄 I thought you changed it by the comment 😅

Reply:
Awesome, thanks👍🏽